### ISSUE TITLE
[add] new datasets - anime characters

### ISSUE THREAD

**Issue by sarathm09** [sarathm09/test-datasets#6]

**Comment by jeannienguyen** [comment]
@sarathm09 Do you need the info pulled from anywhere specifically?

**Comment by sarathm09** [comment]
No, nothing specific as such. The intent of this repo is to provide apis for such data so that anyone can use the data about the stuff they like. So you can provide any data that you like and that you feel others would benefit from 🙂
